pkg/v1/tkg/test/tkgctl/tkgs/tkgs_suite_test.go

```diff
@@ -156,3 +157,34 @@ func createClusterConfigFile(e2eConfig *framework.E2EConfig) string {
 	Expect(err).To(BeNil())
 	return clusterConfigFile
 }
+
+// getCalicoCNIClusterClassFile return temporary Cluster Class config file with Calico CNI
+func getCalicoCNIClusterClassFile(e2eConfig *framework.E2EConfig) string {
+	clusterConfigFile := e2eConfig.WorkloadClusterOptions.ClusterClassFilePath
+	if _, err := os.Stat(clusterConfigFile); err != nil {
+		return clusterConfigFile
+	}
+
+	yamlFile, err := os.ReadFile(clusterConfigFile)
+	if err != nil {
+		return clusterConfigFile
+	}
```

Comment:
Wondering if this wouldn't this hide the error as it returns the default cluster config file without calico?

Reply:
Make sense, if it hides error the calico test will become the same as antrea test.. Will update the test to expect this function to succeed.

Reply:
The function has been udpated by checking the return error. Thanks for the suggestion!